data/wpa_supplicant/wpa_supplicant_test.sh

```diff
@@ -2,6 +2,9 @@
 
 function prepare() {
 	modprobe mac80211_hwsim
+	# Ensure there are no other processes running that disturb this test
+	killall wpa_supplicant 2>/dev/null || true
```

Comment:
```suggestion
        systemctl disable --now wpa_supplicant || true
	killall wpa_supplicant 2>/dev/null || true
```

Reply:
wpa_supplicant is started through a systemd service. I thought it is more clear to disable the service.
But we would need to start this service again when using wicked later on!? So I'm actually not 100% sure about it.
What do you think here?
I mean in the long term, we will remove wicked here and just keep the native wpa_supplicant tests.

Reply:
I agree with the suggestion and included it.